### ISSUE TITLE
Add CSS media queries head boxes on small resolutions

### ISSUE THREAD

**Issue by eusonlito** [allinurl/goaccess#410]
In tables and mobile is very complicated to read header values.

For example, add:

```css
@media (max-width: 1024px) {
    .col-1-6 {
        width: 33%
    }
}

@media (max-width: 480px) {
    .col-1-6 {
        width: 50%
    }
}
```

![goaccess-responsive](https://cloud.githubusercontent.com/assets/644551/15120878/18ac5124-1618-11e6-890a-0175576ac60a.png)

**Comment by allinurl** [comment]
Thanks for pointing this out. I'm working on v1.0 - #223 which changes the HTML output. I'll look into this and make sure the upcoming version handles this right.

**Comment by eusonlito** [comment]
Perfect, thanks a lot.

**Comment by allinurl** [comment]
This should be fixed and has been deployed in [version 1.0](https://goaccess.io/). Thanks!
